src/transformer/decode.ts

```diff
@@ -802,6 +833,21 @@ export class Decoder {
     )
   }
 
+  public buildKeywordsContents = (manuscriptKeywords: ManuscriptKeyword[]) => {
+    const p = document.createElement('p')
+    p.classList.add('keywords-list')
+
+    manuscriptKeywords.map((keyword) => {
+      const keywordElement = document.createElement('span')
```

Comment:
Perhaps I'm missing something but It is not clear why we create an html node here and provide a string instead of following the normal approach: creating a node type corresponding to each element using our prosemirror schema and building a content array to be used in createAndFill method. Have a look at how paragraphs are assembled into corresponding sections in [ObjectTypes.Section]. If we will go this way (creating string content) we will have to parse that string every time we need to do something about it, essentially working around the entire prosemirror architecture.

Reply:
Fixed.